test/main.spec.js

```diff
@@ -0,0 +1,14 @@
+import test from 'tape'
+import addTwoNumbers from './../src/main'
```

Comment:
@emmenko [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import) does not support [NODE_PATH natively](https://github.com/benmosher/eslint-plugin-import/issues/130), to make it to work, we need to add some [new settings](https://github.com/benmosher/eslint-plugin-import/issues/130#issuecomment-160416915) in a .eslintrc file. Should we do that at the [eslint-config](https://github.com/commercetools/eslint-config) plugin level or just add the file to the nodejs-boilerplate?
Cc @Siilwyn @PhilippSpo